### ISSUE TITLE
Style attributes in HTML being flagged as errors

### ISSUE THREAD

**Issue by smlombardi** [prettier/stylelint-prettier#25]
**What version of `stylelint`, `prettier` and `stylelint-prettier` are you using?**
```
    "stylelint": "10.0.1",
    "stylelint-config-prettier": "5.2.0",
    "stylelint-prettier": "1.1.0"
```

**Please paste any applicable config files that you're using (e.g. `.prettierrc` or `.stylelintrc` files)**
```
{
  "printWidth": 120,
  "singleQuote": true,
  "useTabs": false,
  "tabWidth": 2,
  "semi": true,
  "bracketSpacing": true
 }
```
```
{
  "plugins": ["stylelint-prettier"],
  "extends": ["stylelint-prettier/recommended"],
  "rules": {
    "at-rule-empty-line-before": null,
    "at-rule-no-unknown": null,
    "declaration-colon-newline-after": null,
    "declaration-empty-line-before": null,
    "max-empty-lines": 4,
    "indentation": 2,
    "no-eol-whitespace": [
      true,
      {
        "ignore": ["empty-lines"]
      }
    ],
    "number-leading-zero": null,
    "no-empty-source": null,
    "rule-empty-line-before": null,
    "selector-list-comma-newline-after": "always-multi-line",
    "selector-list-comma-space-after": "always-single-line",
    "selector-pseudo-element-colon-notation": "double",
    "selector-type-no-unknown": null,
    "string-no-newline": null,
    "no-missing-end-of-source-newline": null,
    "no-descending-specificity": null,
    "no-duplicate-selectors": null,
    "unit-whitelist": ["em", "rem", "s", "px", "%", "deg", "ms", "vh", "vw", "dppx", "dpi", "fr"],
    "prettier/prettier": [
      true,
      {
        "singleQuote": false,
        "tabWidth": 2
      }
    ]
  }
}
```

**What source code are you linting?**
```
 <p class="mb-0"
         style="font-size: 12px;">
        By clicking "Complete My Application" you agree to
        <a class="fake-link">our terms of use</a>,
        <a class="fake-link">privacy</a>
        and
        <a class="fake-link">refund policies</a>.
      </p>
```

**What did you expect to happen?**
The style tag is valid html and should not be flagged

**What actually happened?**
<img width="815" alt="Screen Shot 2019-05-28 at 9 29 14 AM" src="https://user-images.githubusercontent.com/835148/58481905-591a8d80-812b-11e9-8ba6-d6d6ecd53418.png">

**Comment by seangwright** [comment]
I'm seeing the same thing in my templates:

![Annotation 2020-03-18 181544](https://user-images.githubusercontent.com/1382768
/77012549-877c8a00-6944-11ea-993b-c72846ebc52e.jpg)

```json
"stylelint": "^13.2.1",
"stylelint-config-prettier": "^8.0.1",
"stylelint-config-standard": "^20.0.0",
"stylelint-prettier": "^1.1.2",
"stylelint-scss": "^3.15.0",
```

```json
{
  "extends": ["stylelint-config-standard", "stylelint-prettier/recommended"],
  "plugins": [
    "stylelint-scss"
  ],
  "rules": { ... }
}
```

**Comment by jinxinkai** [comment]
I'm seeing the same thing in my code:

```<div class="index-consist" style="padding-bottom: 100vh;">

```

**Comment by JounQin** [comment]
@BPScott Any time to see this problem?

**Comment by matusko89** [comment]
Same here

```
{
    "plugins": ["stylelint-prettier"],
    "extends": ["stylelint-prettier/recommended"],
    "rules": {
        "prettier/prettier": [true, {
            "singleQuote": false,
            "tabWidth": 4
        }]
    }
}

```
```
    "stylelint": "^13.8.0",
    "stylelint-config-prettier": "^8.0.2",
    "stylelint-prettier": "^1.1.2",
```

**Comment by JounQin** [comment]
For anyone may be interested, #156 should fix this issue, but @BPScott haven't come to review and publish it.

So I decide to publish a new package [`@rxts/styelint-prettier@1.2.0`](https://www.npmjs.com/package/@rxts/stylelint-prettier), once that PR been merged and published, I will deprecate it.

Source code at https://github.com/rx-ts/stylelint-prettier/tree/parser

**Comment by BPScott** [comment]
I've published stylelint-prettier v1.2.0 with support for .component.html files (which use a different parser than regular .html files, which is why I suspect you were seeing these issues as v1.1.0 ways already ignoring .html files) and .svelte files.

Thanks to @JounQin for his initial investigation into this.
